server/docs/source/index.html.md

```diff
@@ -1490,6 +1491,7 @@ Let's Encrypt domain authorization management for certificate handling.
 - `created`: authorization has been created, no firther actions yet taken
 - `deploying`: The related tls-sni certificate is currently being deployed to linked service. Only valid for tls-sni type of authorizations
 - `deploy_error`: The deployment of the linked service has errored out, more details can be found from the linked services event logs
+- `expired`: The domain authorization can no longer be used to request a certificate, the domain must be re-authorized
```

Comment:
Should this be `expires_at` ?

Reply:
That's the `status` value, not the JSON `expired_at` field.

Reply:
Right, but it seems that not all object attributes are documented.

Reply:
Not sure what you mean - only the `challenge_opts` and `status` fields have some kind of extra docs apart from the JSON example?

I guess the domain authz/certificates are missing the attribute/description table.